src/api/services/publishDocuments.js

```diff
@@ -26,6 +30,7 @@ export const versionTransformerDecorator = (
                 publicationDate,
             },
         ],
+        ...hiddenResource,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why not return `hiddenResource` instead of `...hiddenResource`? I'm worried some fields in `hiddenResource` could overlap with other fields from the object, like `uri` for instance...

Reply:
Actually I get why you need to 'merge' the properties in the object (to populate the `removedAt` field at least), but my concern about the `uri` field still applies

Reply:
Resolved by moving `...hiddenResource` at the top